client/concordclient/include/client/concordclient/concord_client.hpp

```diff
@@ -9,6 +9,8 @@
 // terms and conditions of the sub-component's license, as noted in the LICENSE
 // file.
 
+#pragma once
```

Comment:
I was of the impression the Google C++ style guide called for define guards (https://google.github.io/styleguide/cppguide.html#The__define_Guard) over `#pragma once` (https://google.github.io/styleguide/cppguide.html#Windows_Code). Do we officially diverge from the style guide on this particular matter?

Reply:
Pretty much everyone in concord-bft just uses `#pragma once` for simplicity. This works due to the recent, and limited version of compilers we support.

Reply:
Good point @upshaw-alex, I went with the convenient way as most do and as @andrewjstone pointed out. "Rules are rules" but in this case I think we should allow `#pragma once` specifically in our `CONTRIBUTING.md` given that I cannot think of or haven't found any issues with it that would be solved by include guards. I will prepare a change.